neo4jgraph.py

```diff
@@ -0,0 +1,211 @@
+from neo4j import GraphDatabase
```

Comment:
Please add a docstring describing the contents and usage of the module.

go/pystyle#module-docs